overlord/configstate/configcore/network.go

```diff
@@ -76,11 +80,13 @@ func handleNetworkConfiguration(tr config.Conf) error {
 		return err
 	}
 
-	// load the new config into the kernel
-	if len(changed) > 0 || len(removed) > 0 {
-		output, err := exec.Command("sysctl", "-w", sysctl).CombinedOutput()
-		if err != nil {
-			return osutil.OutputErr(output, err)
+	if opts == nil || opts.RootDir == "" {
```

Comment:
we control the code in the case that there is no opts,  we know we pass in nil and not an empty struct so we don't need the || and RootDir check

Reply:
Right, done (in all places).